README.md

```diff
@@ -318,6 +318,9 @@ introduced between each file).
 
 ### head([{'-n', \<num\>},] file [, file ...])
 ### head([{'-n', \<num\>},] file_array)
+Available options:
```

Comment:
@ariporad As I was digging through, I saw that the documentation had an error (it was missing the available option). You're correct, these changes are unrelated to the refactor.